.github/workflows/ci.yaml

```diff
@@ -45,6 +45,9 @@ jobs:
       - name: Test
         run: npx nx affected --target=test --verbose --parallel=1 --base=remotes/origin/main --head=HEAD --ci
 
+      - name: Benchmark
+        run: npm run benchmark
+
```

Comment:
Can you move that to another CI job called `benchmark` or so in this file? I assume that we extend the functionality of this benchmark system soon and then we don't clutter the normal CI job with it.